jakarta/quarkus3.yml

```diff
@@ -227,6 +229,64 @@ recipeList:
       oldFullyQualifiedTypeName: javax.annotation.sql.DataSourceDefinitions
       newFullyQualifiedTypeName: jakarta.annotation.sql.DataSourceDefinitions
 
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.JavaxAuthenticationMigrationToJakartaAuthentication
+displayName: Migrate deprecated `javax.security.auth.message` packages to `jakarta.security.auth.message`
+description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
+tags:
+  - authentication
+  - security
+  - javax
+  - jakarta
+
+recipeList:
+  - org.openrewrite.maven.AddDependency:
+      groupId: jakarta.authentication
+      artifactId: jakarta.authentication-api
+      version: 2.x
+      onlyIfUsing: javax.security.auth.message..*
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: jakarta.authentication
+      artifactId: jakarta.authentication-api
+      newVersion: 2.x
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: javax.security.auth.message
+      newPackageName: jakarta.security.auth.message
+      recursive: true
+  - org.openrewrite.maven.RemoveDependency:
+      groupId: javax.security.auth.message
+      artifactId: javax.security.auth.message-api
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.JavaxAuthorizationMigrationToJakartaAuthorization
+displayName: Migrate deprecated `javax.security.jacc` packages to `jakarta.security.jacc`
+description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
+tags:
+  - authorization
+  - security
+  - javax
+  - jakarta
+
+recipeList:
+  - org.openrewrite.maven.AddDependency:
+      groupId: jakarta.authorization
+      artifactId: jakarta.authorization-api
+      version: 2.x
+      onlyIfUsing: javax.security.jacc..*
```

Comment:
@tkvangorder I've changed this line from `jakarta.security.jacc..*` to `javax.security.jacc..*`. It makes more sense, right? I mean, add the new dependency when in the project, there's used an old reference to the package.

Reply:
@gsmet Yep, this parameter was even changed on the OpenRewrite side. You can check it here: https://github.com/openrewrite/rewrite-migrate-java/pull/167/files

Reply:
Same remark as above.